src/Regridder.jl

```diff
@@ -633,5 +633,49 @@ function cgll2latlonz(field; DIR = "cgll2latlonz_dir", nlat = 360, nlon = 720, c
     return new_data, coords
 end
 
+"""
+    truncate_dataset(date0, timeStart, timeEnd, datafile)
+
+Truncates given data set to only regrid the dates that are used in the simulation
```

Comment:
It's more accurate to avoid saying anything about regridding in this function, since we're just truncating the dataset but not regridding yet. Maybe you could say something like `Truncates given dataset, and constructs a new dataset containing only the dates that are used in this simulation`